src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
@davidtheclark why? it is 3rd column.
```
(1 column).(2 column)a(3 column)
```
`a` - it is 1 line and 3 column. Right?